### PR TITLE
Exceptions 

### PR DESCRIPTION
**Pull request by KarlisTi**

### CODE REVIEW

File: src/main/java/io/codelex/Exceptions/practice/Exercise1.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package io.codelex.Exceptions.practice;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Exercise1 {
```
Comment (NilsKula) [inline src/main/java/io/codelex/Exceptions/practice/Exercise1.java:6]: missing quit logic

File: src/main/java/io/codelex/Exceptions/practice/Exercise2.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    public static void main(String[] args) {
+
+        methodA();
+
+    }
+
+    public static void methodA() throws ArithmeticException {
+        try {
+            methodB();
+        } catch (ArithmeticException e) {
```
Comment (NilsKula) [inline src/main/java/io/codelex/Exceptions/practice/Exercise2.java:16]: `After each try{} block put a catch{} block which catches the exception, prints a stack trace, and throws the exception object to its caller`

**Review by NilsKula — CHANGES_REQUESTED** [review]

File: src/main/java/io/codelex/Exceptions/practice/Exercise2.java (hunk 2 above)
Comment (KarlisTi) [reply src/main/java/io/codelex/Exceptions/practice/Exercise2.java:15]: Ieliku, lai met kļūdu tam saucējam

**Review by KarlisTi — COMMENTED** [review]
Pārskatīju

**Review by NilsKula — APPROVED** [review]

**Pull request merged**